command/root.go

```diff
@@ -107,7 +110,9 @@ func BasicClient() (*api.Client, error) {
 	}
 	opts = append(opts, api.AddHeader("User-Agent", fmt.Sprintf("GitHub CLI %s", Version)))
 	if c, err := context.ParseDefaultConfig(); err == nil {
-		opts = append(opts, api.AddHeader("Authorization", fmt.Sprintf("token %s", c.Token)))
+		if token, err := c.Get(defaultHostname, "oauth_token"); err == nil {
```

Comment:
In which case would there be an error here, and should we keep tolerating it?

Reply:
it would have to be something messed up in the config and we shouldn't tolerate it; changed to fail